automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookClusterMonitoringSpec.scala

```diff
@@ -286,7 +287,7 @@ class ClusterMonitoringSpec extends FreeSpec with LeonardoTestUtils with Paralle
         withNewCluster(project, request = ClusterRequest()) { cluster =>
```

Comment:
I think we should refactor/relocate this test. Jupyter lab installation is tested in`LabSpec` so we can get rid of that. We should just test for loc/delocalization.